Azure/sonic-buildimage
# Add support get_transceiver_change_event in as9716-32d

**- What I did**
Add support get_transceiver_change_event in as9716-32d
**- How I did it**
Implement code to cpld and sfputil.py for get_transceiver_change_event in as9716-32d
**- How to verify it**
Write test.py to read get_transceiver_change_event() and insert port to check whether port evt change.
Log as blow,
Got port-1 inserted event
Got port-31 removed event
Got port-31 inserted event

**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
